lib/cn/node_split.c

```diff
@@ -536,6 +537,7 @@ cn_split(struct cn_compaction_work *w)
         return merr(ENOMEM);
 
     wq = cn_get_io_wq(w->cw_tree->cn);
+    inflight = w->cw_kvset_cnt;
```

Comment:
I didn't know that we can operate on an _Atomic type like its non-atomic counterpart, i.e., without using atomic_load/store/fetch.. interfaces..

Although handy, a downside with this approach is that it reduces code readability as I now need to check the declaration to know whether we are operating on an atomic or not.